addons/project_timesheet_holidays/models/hr_employee.py

```diff
@@ -17,6 +17,13 @@ def create(self, vals):
         self._create_future_public_holidays_timesheets(employees)
         return employees
 
+    def _delete_current_public_holidays_timesheets(self):
```

Comment:
detail: method could be positioned at the end (it's not always respected, but in the [guideline](https://www.odoo.com/documentation/16.0/contributing/development/coding_guidelines.html#symbols-and-conventions) the business method are grouped together, separately from CRUD method